charts/ks-devops/templates/_helpers.tpl

```diff
@@ -30,6 +29,19 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Common labels for ks-devops
```

Comment:
Those labels will be added by helm automatically. We don't have to deal with it manually.